pom.xml

```diff
@@ -88,7 +88,15 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>ssh-credentials</artifactId>
-            <version>1.13</version>
+            <version>1.14</version>
+        </dependency>
+
+        <!-- test dependencies -->
+        <dependency>
+            <groupId>org.apache.sshd</groupId>
+            <artifactId>sshd-core</artifactId>
+            <version>2.1.0</version>
```

Comment:
just out of curiosity: is this version the same that is used in the baseline?

Reply:
Jenkins core removed this as a default test dependency. The version previously in use was 0.x. I had to refactor some test code here to use the updated package names (and a couple class names, too). By updating this dependency, this also fixes the previous test failure without having to change much about the test (though I did get rid of the embedded key in the test). As a bonus, this test now verifies that your crypto setup is correct, too, since it uses RSA 4096 which comes with Bouncycastle I believe.